src/translations/en.json

```diff
@@ -417,6 +417,9 @@
         "add_device_id": "Choose device",
         "add_entity_id": "Choose entity"
       },
+      "config-entry-picker": {
+        "config_entry": "Config entry"
```

Comment:
"Config entry" is not a term we use in general towards the end-user. Should this be "Integration entry"? Or "Integration instance" or something similar?

Reply:
The tab in config is called Integrations. I guess we don't distinguish between Integration entries. I think just Integration is fine.